javascript-light-bulb/main.js

```diff
@@ -21,13 +21,13 @@ $button.addEventListener('click', handleClick);
 var buttonOn = false;
 
 function handleClick(event) {
-  if (buttonOn === false) {
-    buttonOn = true;
-    $button.className = 'bt-dark';
-    $background.className = 'bg-dark';
-  } else if (buttonOn === true) {
+  if (buttonOn === true) {
```

Comment:
There's no need for the comparison here, this line can be just `if(buttonOn){`

Reply:
ohhh i see